pyproject.toml

```diff
@@ -1,13 +1,14 @@
 [tool.poetry]
-name = "fastapi_gateway"
-version = "0.0.3"
+name = "fastapi_gateway_ultra"
```

Comment:
Also describe why you made the decision to change the name of the package and made a lot of edits about it, if it is appropriate and will bring some benefit, I can consider your suggestion, but for now it seems pointless to me.